google/resource_container_node_pool_test.go

```diff
@@ -550,3 +592,25 @@ resource "google_container_node_pool" "np_with_node_config_scope_alias" {
 		oauth_scopes = ["compute-rw", "storage-ro", "logging-write", "monitoring"]
 	}
 }`, acctest.RandString(10), acctest.RandString(10))
+
+var testAccContainerNodePool_withManagement = fmt.Sprintf(`
+resource "google_container_cluster" "cluster" {
+	name = "tf-cluster-nodepool-test-%s"
+	zone = "us-central1-a"
```

Comment:
nit: mind running `terraform fmt` on this file to align all the `=`s?

Reply:
Unfortunately `vim-terraform` doesn't autodetect nested strings within go files.  😛 
Note that If i visually select each block and pipe out to `terraform fmt` (i.e. `:'<,'>!terraform fmt -`), I end up with two leading spaces, rather than tabs, along with various other format changes for pretty much every single nested block within the file, which is actually what I'd expect from `terraform fmt`.  Should we do this for all the tests uniformly in a separate PR?  I'll push a separate commit doing so for the individual file here, just to illustrate, which I can discard (or not).

Reply:
`:set expandtab`|`:set noexpandtab` and `:GoFmt` both appear to have no effect, either

Reply:
Ah yeah, we don't really have a consistent story on whether we should use tabs or spaces for configs within tests. I don't really have an opinion either way, I just want the nice alignment :)
I do think though that we should either use spaces for all tests (in a given file) or tabs for all, so for now it probably makes sense to discard the commits and align by hand

Reply:
Yeah, that seems sane.  I'm generally not a huge fan of owning hundreds of lines of code I don't necessarily understand for the sake of formatting.  😛